### DISCUSSION TITLE
The future of botfront

### DISCUSSION THREAD

**Discussion by WeberJulian** [botfront/botfront#996] [General]
First of all, I really like what you've done with botfront. You filled a void that really needed filling in the market.
That's why the company that I represent want to use botfront in production.
I read the [blog-post](https://botfront.io/blog/botfront-acquisition-botfront-1-0-and-the-community/) announcing your acquisition by Dialogue and was really grateful that you decided to opensource the enterprise features. (btw the link to it on your website is broken).

However I have concerns about the botfront project. Now that you have been acquired, what does it mean for the project mid/long term ? Do you still accept suggestions ? Answer questions in issues ? Will you continue to maintain repos like rasa-webchat (that [doesn't work](https://github.com/botfront/rasa-webchat/pull/346) with the latest botfront version). Do you still plan to integrate the latest rasa releases ?

I definitely don't wanna sound entitled because I understand that you guys may have new priorities but I would be grateful for some visibility on what you've planed for the future of botfront before spending R&D time on putting a project in production.

Thank you for your time

**Comment by m13** [comment]
Any update here? We are also considering using botfront in a production project. Thanks!
